src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public abstract class Figure implements Draw, Area {
+    private String color;
+
+    public Figure(String color) {
+        this.color = color;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    @Override
+    public void print() {
```

Comment:
The method 'print' should be conceptually similar to the methods in the interface it belongs to. If 'Draw' and 'Area' are separate concerns, they should not be in the same interface.